src/applications/gi/components/vet-tec/VetTecHeadingSummary.jsx

```diff
@@ -17,6 +17,15 @@ export const VetTecHeadingSummary = ({ institution }) => (
       <h1>{institution.name}</h1>
       <div>
         <div className="usa-width-one-half medium-6 small-12 column">
+          <IconWithInfo
+            icon="star vads-u-color--gold"
+            present={
```

Comment:
I don't think this `present` prop is used at all in the component. Seems like you meant to use it similarly to the other `IconWithInfo` in `gi/components/profile/HeadingSummary`, which does hide the icon if `present` is truthy. If that's the case, you just need to add that condition to this `IconWithInfo`.

Reply:
Good catch! Change is in.